app-frontend/src/app/components/scenes/projectSceneItem/index.html

```diff
@@ -0,0 +1,58 @@
+<div class="list-group-item">
+  <div class="list-item-left-actions"
+       tooltips
+       tooltip-template="Image is already in {{$ctrl.scene.inLayer ? 'layer' : 'project'}}"
+       tooltip-size="small"
+       tooltip-class="rf-tooltip shrink"
+       tooltip-side="right"
+       tooltip-hidden="{{!$ctrl.scene.inLayer && !$ctrl.scene.inProject}}"
+       ng-if="!$ctrl.processing"
+  >
+    <label class="checkbox"
+           ng-class="{
+                  active: $ctrl.selected || $ctrl.scene.inLayer,
+                  'partial-active': $ctrl.scene.inProject
```

Comment:
I think it makes sense to disallow adding scenes already in the layer, but probably it's ok to add scenes to layer A, even if it is already in layer B? But this is my opinion.

Reply:
I don't prevent users from adding them to multiple layers - the checkbox just has a third state to indicate that it is in other layers